### PR TITLE
Adding Figure Summary + Legends

### LINKED ISSUE #49: Write Figure Titles and Figure Legends

### PR DESCRIPTION
**Pull request by gwaygenomics**
closes #49 

Here I add a `README.md` that summarizes the results in order of analysis and adds figure legends. I convert a select number figures from `pdf` to `png` for easy viewing in the README file. The `pdf` files should be used for final figures.

### CODE REVIEW

File: figures/README.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+<object data="https://github.com/greenelab/pdx_exomeseq/blob/master/figures/upset/upset_sample_001.pdf" type="application/pdf" width="700px" height="700px">
+    <embed src="https://github.com/greenelab/pdx_exomeseq/blob/master/figures/upset/upset_sample_001.pdf">
+           This browser does not support PDFs. Please download the PDF to view it: <a href="https://github.com/greenelab/pdx_exomeseq/blob/master/figures/upset/upset_sample_001.pdf">Download PDF</a>.</p>
+    </embed>
+</object>
+
+**Figure Legend.** Exploring the overlap of COSMIC variants across PDX passages.
+The UpSet plots ([Conway et al. 2017](https://doi.org/10.1093/bioinformatics/btx364)) depict the number of variants in different sets of the data.
+The total overlapping set is highlighted in orange.
+
```
Comment (huqiwen0313) [inline figures/README.md]: Add some description for the bottom figure ?

File: figures/README.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+## Oncoprint Diagram of Mutated Genes across Samples
+
+### Replicates
+
+<object data="https://github.com/greenelab/pdx_exomeseq/blob/master/figures/oncoprint_replicates.pdf" type="application/pdf" width="700px" height="700px">
+    <embed src="https://github.com/greenelab/pdx_exomeseq/blob/master/figures/oncoprint_replicates.pdf">
+           This browser does not support PDFs. Please download the PDF to view it: <a href="https://github.com/greenelab/pdx_exomeseq/blob/master/figures/oncoprint_replicates.pdf">Download PDF</a>.</p>
+    </embed>
+</object>
+
```
Comment (huqiwen0313) [inline figures/README.md]: The bottom legend is a little small

File: figures/README.md (hunk 1 above)
Comment (gwaygenomics) [reply figures/README.md]: ah, good point. will add

File: figures/README.md (hunk 2 above)
Comment (gwaygenomics) [reply figures/README.md]: I agree. I will make the updates in a future PR. See #64 for details. Thanks!

**Pull request merged**